nss-day-cohort-47/tabloid-mvc-cookie-crisp
# V2 edit tag bk

# Description

Edit methods for tag section are complete // full tag CRUD is done

## Type of change

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions

Navigate to tag section and edit then save an existing tag object.  It should update the database. Please check create and delete options also

# Checklist:

- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
